src/Components/Grants/GrantShowOverview.jsx

```diff
@@ -1,29 +1,140 @@
-/* eslint-disable no-unused-vars */
-import React, { useState } from "react";
+import React, { useEffect } from "react";
 import { useQuery, useMutation } from "react-query";
 import { useParams } from "react-router-dom";
+import Button from "../design/Button/Button";
 import Container from "../design/Container/Container";
-import SectionListItem from "../Sections/SectionListItem";
+import Hero from "../design/Hero/Hero";
+import { useCurrentOrganization } from "../../Contexts/currentOrganizationContext";
+import * as GrantsService from "../../Services/Organizations/GrantsService";
+import * as SectionsService from "../../Services/Organizations/Grants/SectionsService";
+import countSectionWords from "../../Helpers/countSectionWords";
+import { SortableItem } from "./SortableItem";
+import { Item } from "./Item";
+import CurrentOrganizationLink from "../Helpers/CurrentOrganizationLink";
+import {
+  SortableContext,
+  verticalListSortingStrategy,
+} from "@dnd-kit/sortable";
+import { DragOverlay } from "@dnd-kit/core";
+
 import "./GrantShowOverview.css";
-import SortableElement from "../Elements/SortableElement";
 
-export default function GrantShowOverview({ grant }) {
+function countTotalSectionsWords(sections = []) {
+  return sections?.reduce(
+    (total, section) => total + countSectionWords(section),
+    0
+  );
+}
```

Comment:
I think this function already exists in `GrantsShow.jsx`. Could we move it into the helpers directory and reuse it here?